diff --git a/node_modules/@massif/ktb-data/index.js b/node_modules/@massif/ktb-data/index.js
index 440d477..6be1d67 100644
--- a/node_modules/@massif/ktb-data/index.js
+++ b/node_modules/@massif/ktb-data/index.js
@@ -10,7 +10,7 @@ const data = {
   lcp_manifest: require('./lib/lcp_manifest.json'),
   // manufacturers: require('./lib/manufacturers.json'),
   // mods: require('./lib/mods.json'),
-  pilot_gear: require('./lib/pilot_gear.json'),
+  // pilot_gear: require('./lib/pilot_gear.json'),
   // reserves: require('./lib/reserves.json'),
   // sitreps: require('./lib/sitreps.json'),
   // skills: require('./lib/skills.json'),
diff --git a/node_modules/@massif/ktb-data/package.json b/node_modules/@massif/ktb-data/package.json
index 7633c91..24d8b2c 100644
--- a/node_modules/@massif/ktb-data/package.json
+++ b/node_modules/@massif/ktb-data/package.json
@@ -3,7 +3,6 @@
   "version": "1.2.3",
   "description": "Player-facing COMP/CON data for the 'Karrakin Trade Baronies', a setting for the LANCER TTRPG",
   "main": "index.js",
-  "type": "module",
   "scripts": {
     "build": "node ./scripts/build.js",
     "cleanup": "node ./scripts/cleanup.js",
