diff --git a/node_modules/@massif/long-rim-data/index.js b/node_modules/@massif/long-rim-data/index.js
index bcdd9e5..7a6a314 100644
--- a/node_modules/@massif/long-rim-data/index.js
+++ b/node_modules/@massif/long-rim-data/index.js
@@ -9,7 +9,7 @@ const data = {
   frames: require('./lib/frames.json'),
   // manufacturers: require('./lib/manufacturers.json'),
   mods: require('./lib/mods.json'),
-  pilot_gear: require('./lib/pilot_gear.json'),
+  // pilot_gear: require('./lib/pilot_gear.json'),
   // reserves: require('./lib/reserves.json'),
   // sitreps: require('./lib/sitreps.json'),
   // skills: require('./lib/skills.json'),
diff --git a/node_modules/@massif/long-rim-data/package.json b/node_modules/@massif/long-rim-data/package.json
index 8162b48..38e25d1 100644
--- a/node_modules/@massif/long-rim-data/package.json
+++ b/node_modules/@massif/long-rim-data/package.json
@@ -3,7 +3,6 @@
   "version": "1.2.1",
   "description": "Player-facing COMP/CON data for 'The Long Rim', a setting for the LANCER TTRPG",
   "main": "index.js",
-  "type": "module",
   "scripts": {
     "build": "node ./scripts/build.js",
     "cleanup": "node ./scripts/cleanup.js",
